cogs5e/models/initiative/combat.py

```diff
@@ -32,6 +33,9 @@ def __init__(self, channel_id, message_id, dm_id, options, ctx,
         self._turn = turn_num
         self._current_index = current_index
         self.ctx = ctx
+        if metadata is None:
+            metadata = {}
```

Comment:
Convention is to put all mutable default arguments at the top (in this case, right under `if combatants is None`)